### ISSUE TITLE
Prefab link on scene object still exists after prefab removed

### ISSUE THREAD

**Issue by TheJustLink** [FlaxEngine/FlaxEngine#1017]
**Steps to reproduce:**
1. Create object on scene
2. Create prefab
3. Remove prefab in assets
4. Prefab link on scene object still exists and we get warnings in output log (last warning line showed after save the scene):
![image](https://user-images.githubusercontent.com/29580493/233090495-4004de67-540e-4d6f-8812-d1328ba9f8e6.png)

**Flax version:** [1.5.6339](https://github.com/FlaxEngine/FlaxEngine/releases/tag/1.5.6339)

**Comment by RuanLucasGD** [comment]
@TheJustLink  are you sure this is a problem? In Unity this works similary, it's signaled that the prefab does not exist, but the object reamins in the scene.

**Comment by TheJustLink** [comment]
@RuanLucasGD, Yes, the main problem here is that it's not visible or noticeable. It should repaint prefabs without a link in some red color.

**Comment by mafiesto4** [comment]
The thing is that serialized prefab instance data inside the scene file is not enough to restore that deleted object (eg. even object type is missing, we just store prefabId, objectId and overridden properties infos).

We could highlight prefab objects in red when that scene is opened or even better search for usages of that prefab and show the asset reference removal dialog like UE4 does:
![image](https://user-images.githubusercontent.com/3464853/236669890-0febfd1d-5a1f-4a7b-91c0-c532a71fa7d0.png)

**Comment by mafiesto4** [comment]
Also here is how the warning message looks when opening scene with missing prefab asset:
```
[ 00:00:05.499 ]: [Warning] Invalid or missing asset (da899c39489522ad54ea31a27f43fa77, FlaxEngine.Prefab).
[ 00:00:05.499 ]: [Warning] Missing prefab with id=da899c39489522ad54ea31a27f43fa77.
[ 00:00:05.499 ]: [Warning] Failed to deserialize scene object from data: {
	"ID": "2a2c690f484e23295b8f67b9a6dde20e",
	"PrefabID": "da899c39489522ad54ea31a27f43fa77",
	"PrefabObjectID": "950c63204c997927f95fe18e161dbba7",
	"ParentID": "cf6abd00481bc8b4b65e7f889ec93f9e",
	"Name": "tmpPrefab",
	"Transform": {
		"Translation": {
			"X": 849.455810546875,
			"Y": 49.99993896484375,
			"Z": 1508.2344970703126
		}
	}
}
[ 00:00:05.499 ]: [Warning] Parent actor of the missing object: Scene
```

**Comment by ElevenArt** [comment]
I mentioned this issue in my [Prefab thread on Discord](https://discord.com/channels/437989205315158016/1270707042834845747/1324872926251712633)
